src/main/scala/io/opentargets/etl/backend/Evidence.scala

```diff
@@ -465,7 +468,15 @@ object Evidence extends LazyLogging {
 
     val lut = broadcast(
       diseases
-        .select(col("id").as("dId"), $"name".as("diseaseLabel"))
+        .select(
+          col("id").as("efoId"),
+          explode(
+            concat(
+              array(col("id")),
+              coalesce(col("obsoleteTerms"), typedLit(Array.empty[String]))
+            )
+          ).as("did")
+        )
         .orderBy($"dId".asc)
         .repartition($"dId")
     )
```

Comment:
Is line 478 a typo? You select columns 'efoId' and 'did' from diseases, but then order by 'dId'.